### ISSUE TITLE
Intellisense not working for new files

### ISSUE THREAD

**Issue by asibahi** [ionide/ionide-vscode-fsharp#81]
Hello. I am on Windows 10 ,VS Code 1.2.1 and Ionide 1.9.2

F# Intellisense works very well (if too eager perhaps) when I am editing F# projects saved from Visual Studio, or having a folder open. 

However, when I open a new file and set the language to F# , Intellisense doesn't work. I looked up the previous issues with similar issues with Intellisense but I couldn't understand how they resolved the problem. I cannot tell  if this is a missing feature from Ionide or if I am just missing something simple that would enable it.

This is what it looks like when I press Ctrl+Space. I even saved the file as .fsx and opened it again. Same issue.

![capture](https://cloud.githubusercontent.com/assets/17417266/16276176/f3474c5c-38b5-11e6-97a4-7648f0818009.JPG)

**Comment by rvacaflor** [comment]
I am having the exact same problem with any f# script I have. I do not get any type of Intellisense. I did not test non f# scripts.

I have 2 other PCs that I installed VS Code, on both of them intellisenses works fine for f#.

The only difference that I could find between them is that on the 2 PC that Intellisense works I also have VS 2013 and VS 2015 community and pro installed.

The computer that intellisence is not working I only used the required software for ionide

- Visual F# Tools 3.1.2
- Microsoft Build Tools 2013

can there some updated that I am missing I believe that intellisence was working and them something happened and it is no longer working.
